catalog/core/catalog-core-commands/src/main/resources/bin/catalog-export

```diff
@@ -8,11 +8,28 @@
 # See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
 # <http://www.gnu.org/licenses/lgpl.html>.
 
-usage() { echo "Usage: $0 <-d directory> <-z zipname> <-k keystore location> [-a alias]" 1>&2; exit 1; }
+usage() {
+	echo "SYNTAX ";
+	echo "\t$0 <-d directory> <-z zipname> <-k keystore> [-a alias] [-c cql-expr]" 1>&2;
+	echo "";
+	echo "ARGUMENTS";
+	echo "\t-d directory : The absolute path of the directory to export the metacard archive. Directory path must exist.";
+	echo "\t-z zipname   : The name of the archive file to be generated. File in directory will be overwritten if it already exists.";
```

Comment:
Maybe "If a file with this name already exists in the directory, it will be overwritten"?